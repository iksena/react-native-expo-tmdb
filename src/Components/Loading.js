import { Heading, HStack, Spinner } from 'native-base';

function Loading() {
  return (
    <HStack space={2} my={2} justifyContent="center">
      <Spinner accessibilityLabel="Loading posts" />
      <Heading color="primary.500" fontSize="md">
        Loading
      </Heading>
    </HStack>
  );
}

export default Loading;
