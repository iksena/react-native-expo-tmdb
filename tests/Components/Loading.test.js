import { render } from '../test-utils';
import Loading from '../../src/Components/Loading';

describe('Loading component', () => {
  describe('#render', () => {
    it('should render component', () => {
      const { toJSON } = render(<Loading />);

      expect(toJSON()).toMatchSnapshot();
    });
  });
});
