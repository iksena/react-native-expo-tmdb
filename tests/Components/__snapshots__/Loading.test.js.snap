// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component #render should render component 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    dataSet={Object {}}
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "center",
        "marginBottom": 8,
        "marginTop": 8,
      }
    }
  >
    <ActivityIndicator
      accessibilityLabel="loading"
      accessible={true}
      animating={true}
      color="#06b6d4"
      hidesWhenStopped={true}
      size="small"
      style={
        Array [
          Array [
            Object {},
            Object {
              "accessibilityLabel": "Loading posts",
              "dataSet": Object {},
            },
          ],
          undefined,
        ]
      }
    />
    <View
      dataSet={Object {}}
      style={
        Object {
          "width": 8,
        }
      }
    />
    <Text
      dataSet={Object {}}
      style={
        Object {
          "backgroundColor": undefined,
          "color": "#06b6d4",
          "fontFamily": undefined,
          "fontSize": 16,
          "fontStyle": "normal",
          "fontWeight": "700",
          "letterSpacing": 0,
          "lineHeight": 20,
          "textDecorationLine": undefined,
        }
      }
    >
      Loading
    </Text>
  </View>
</RNCSafeAreaProvider>
`;
